### PR TITLE
Issue #834 Type of Registration

### PR DESCRIPTION
**Pull request by bryan-gilbert**
Was able to set a value into the select during unit tests so we've reduced our skips from 3 to 2

### CODE REVIEW

**Comment by bryan-gilbert** [comment]
image

As discussed this task just adds the full range of types into the select dropdown.  Note the length of text in some of these options.

**Comment by bryan-gilbert** [comment]
Note this resolves 834 not 812.

File: ppr-ui/src/financing-statement/financing-statement-type.ts [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
-  SALE_OF_GOODS
-  FORESTRY CONTRACTOR LIEN
-  FORESTRY CONTRACTOR CHARGE
-  FORESTRY SUBCONTRACTOR CHARGE
-  TAX LIEN
-  LAND TAX
-  NOTICE OF MARRIAGE ...
-  */
+  SECURITY_AGREEMENT = 'PPSA Security Agreement',
+  REPAIRERS_LIEN = 'Repairer\'s Lien',
```
Comment (jguertin) [inline ppr-ui/src/financing-statement/financing-statement-type.ts:3]: In Python we use `'` except where the string contains `'`, then we use `"`. Should we do the same in TS?

**Review by jguertin — APPROVED** [review]

File: ppr-ui/src/financing-statement/FinancingStatement.vue [hunk 2]
```diff
@@ -97,7 +98,18 @@ export default createComponent({
...
+    // Create the set of text value pairs for the Type select
+    interface SelectItem {
+      text: string;
+      value: string;
+    }
+    const fsTypes: SelectItem[] = []
+    const fsTypeLabels: string[] = Object.values(FinancingStatementType)
+    Object.keys(FinancingStatementType).forEach((key: string, index: number): void => {
+      fsTypes.push({ text: fsTypeLabels[index], value: key })
+    })
```
Comment (WalterMoar) [inline ppr-ui/src/financing-statement/FinancingStatement.vue:111]: Curious - is this something that we may want outside the component, for re-use? I don't think we need it here, but possibly for the other enums.

File: ppr-ui/tests/unit/financing-statement/FinancingStatement.spec.ts [hunk 3]
```diff
@@ -49,17 +49,22 @@ describe('FinancingStatmentContainer.vue', (): void => {
...
 
-    // Skip until can figure out how to inject value into a vuetify select
-    it.skip('@input - type change should be emitted', async (): Promise<void> => {
+    it('@input - type change should be emitted', async (): Promise<void> => {
       const properties = ref({ editing: true, value: new FinancingStatementModel() })
       const wrapper: Wrapper<Vue> = mount(FinancingStatement, { propsData: properties.value, vuetify })
-
-      wrapper.get('input[name="typeInput"]').setValue('REPAIRERS_LIEN')
+      const sel = wrapper.get('input[data-test-id="FinancingStatement.type.select"]')
+      sel.setValue('REPAIRERS_LIEN')
```
Comment (WalterMoar) [inline ppr-ui/tests/unit/financing-statement/FinancingStatement.spec.ts:56]: So that's all it was?! Nice work.

**Review by WalterMoar — APPROVED** [review]

**Pull request merged**